bin/live/pycbc_live_single_trigger_fits

```diff
@@ -79,6 +90,14 @@ cuts.insert_cuts_option_group(parser)
 #Add some input sanitisation
 args = parser.parse_args()
 
+# These options are mutually required or not needed
+prune_options = [args.prune_loudest, args.prune_window,
+                 args.prune_stat_threshold]
```

Comment:
I don't think the threshold should be required. So throw an error if you have loudest and not window, also if you have window or threshold but not loudest.  Should be doable in 3 lines or so ..

Reply:
It is required after discussion above